core/src/main/scala/cats/Eval.scala

```diff
@@ -310,6 +310,30 @@ private[cats] trait EvalInstances extends EvalInstances0 {
       }
     }
 
+  implicit val catsReducibleForEval: Reducible[Eval] =
```

Comment:
It occurs to me that such a `Traverse` instance can be defined for any comonad, so it it's probably a known thing with some known properties to people who know more than I do :)